tools/pepsi/src/power_off.rs

```diff
@@ -20,14 +20,15 @@ pub struct Arguments {
 
 pub async fn power_off(arguments: Arguments) -> Result<()> {
     if arguments.all {
-        let addresses = HARDWARE_IDS
-            .keys()
-            .map(|&nao_number| async move {
-                let host = number_to_ip(nao_number, Connection::Wired)?;
+        let addresses = TEAM
```

Comment:
This means that if people don't rebuild their pepsi after we add a new robot, it won't find the robot.
Maybe load the `team.toml` from file and if it can't be located, use the compiled-in version as a fallback?

Reply:
Yes, but this is a new feature, is it? This PR keeps the same behavior as before. I'd postpone this for a future rework of removing constants alltogether (#1477)